dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/AIConstants.cs

```diff
@@ -1,11 +1,26 @@
 ﻿
-namespace Microsoft.TeamsAI.AI.Action
+namespace Microsoft.TeamsAI.AI
 {
     /// <summary>
-    /// Names of the default action types.
+    /// The constants used by the AI system.
     /// </summary>
-    public class DefaultActionTypes
+    public static class AIConstants
```

Comment:
Samples should be updated as well, e.g. chef sample uses `DefaultActionTypes.FlaggedInputActionName`.

Reply:
Good catch. Updated